src/wmtk/attribute/MeshAttributeHandle.hpp

```diff
@@ -29,6 +29,8 @@ class MeshAttributeHandle
         TypedAttributeHandle<double>,
         TypedAttributeHandle<Rational>>;
 
+    using ValueVariant = std::variant<char, int64_t, double, Rational>;
```

Comment:
i don't see where tis is used

Reply:
i extended extract_mesh_from_tag to support arbitrary tag types - so you need to be able to pass in a variadic input type to avoid a template. Consider this fake function:
```get_isocontour(const MeshAttributeHandle& handle, const MeshAttributeHandle::ValueVariant& value)```
. technically this should support eigen vector types, i'll fix that later